.github/workflows/build.yml

```diff
@@ -19,6 +19,9 @@ jobs:
       - name: Setup
         uses: ./.github/actions/setup
 
+      - name: Build
+        run: pnpm nx run-many --target=build --parallel
```

Comment:
```suggestion
        run: pnpm nx run-many --target=build --parallel 3
```
Meant to put a 3 here

Reply:
beware that foundry seems to have trouble with parallel tasks

Reply:
we prob need this step in release too

Reply:
That's why I added --paralell 3 here even though 3 is the default.  Safer to be explicit in CI.

@holic can foundry run before everything else?   e.g. can the build pipeline be like

pnpm install => pnpm nx run-many --target=prebuild --parallel 1 => pnpm nx run-many --target=build --parallel 3

Reply:
seems like 3 is still too much for forge/solc
<img width="633" alt="image" src="https://user-images.githubusercontent.com/508855/231671653-c8b21a8f-ec59-4da6-a17c-b66f6edf24e1.png">

it might be possible to run all forge tasks first but we'd need to comb through our packages and make some consistent script changes between em, and some do have an autogen step before a forge build step, so we'd need to introduce three total steps: pre-forge build, forge build, post-forge build

how slow does this get if we just do `--parallel 1` everywhere for now?

Reply:
I set parallel false in nx.json so it never runs parallel to start.  That's fixable later though

Reply:
there are also some things (like tsgen) that can be run in parallel to all 3 forge steps, if that helps